File: array/2089_find_target_indices_after_sorting_array/2089_find_target_indices_after_sorting_array.py
class Solution:
    def targetIndices(self, nums: List[int], target: int) -> List[int]:
        result = []
        
        nums.sort()
        for i in range(len(nums)):
            if nums[i] == target:
                result.append(i)
            if nums[i] > target:
                break
        return result

File: array/2089_find_target_indices_after_sorting_array/2089_find_target_indices_after_sorting_array.go
package main

func targetIndices(nums []int, target int) []int {
	result := []int{}
	sort.Ints(nums)
	for i := 0; i < len(nums); i++ {
		if nums[i] == target {
			result = append(result, i)
		}
		if nums[i] > target {
			break
		}
	}
	return result

}
